appveyor.yml

```diff
@@ -8,15 +8,15 @@ environment:
 
   matrix:
 
-    - TOOLCHAIN: "ninja-vs-15-2017-win64-cxx17"
+    - TOOLCHAIN: "ninja-vs-15-2017-win64-cxx17-nonpermissive"
```

Comment:
Please comment it out (it's not working?) instead of changing and add new toolchain to `# Extra {` / `# }` section after standard toolchains.

As an example:
- https://github.com/ingenue/hunter/blob/b79d5b27403eee1aea1c136a2393a6c033378e61/.travis.yml#L73-L110

Reply:
done